lib/beaker-answers/pe_conf.rb

```diff
@@ -162,5 +201,21 @@ def the_host_with_role(role)
       end
       found_hosts.first
     end
+
+    # Returns true if there is any host with the supplied role
+    # Used for optional roles that may or may not be present
+    #
+    # @return True if there is atleast one host with the supplied role
+    def any_host_with_role?(role)
```

Comment:
why did you need to copy this from beaker?

Reply:
uhhh...I didn't know that was a method in beaker. Makes sense though! Originally I was using the_host_with_role method, which broke specs since that will raise an error if there is no role defined. I'll

Reply:
I'm having issues trying to get beaker-answers to inherit from beaker. So for the moment I'm leaving this in while you review my other changes.